examples/03_connectivity/plot_data_driven_parcellations.py

```diff
@@ -165,13 +174,15 @@
 
 # This object uses method='kmeans' for KMeans clustering with 10mm smoothing
 # and standardization ON
-kmeans = Parcellations(method='kmeans', n_parcels=50,
+start = time.time()
+kmeans = Parcellations(method='kmeans', n_parcels=2000,
                        standardize=True, smoothing_fwhm=10.,
                        memory='nilearn_cache', memory_level=1,
                        verbose=1)
 # Call fit on functional dataset: single subject (less samples)
 kmeans.fit(dataset.func)
-print("KMeans 50 clusters: %.2fs" % (time.time() - start))
+print("KMeans 2000 clusters: %.2fs" % (time.time() - start))
```

Comment:
I did not compare with the other method since KMeans is not really comparable with the agglomerative methods but I find it still interesting to be aware of the timing. I can remove it or say a word on it. What is the best in your opinion ?

Reply:
I'd remove it.